ibis/mapd/operations.py

```diff
@@ -324,6 +324,13 @@ def _set_literal_format(translator, expr):
     return '({})'.format(', '.join(formatted))
 
 
+def _cross_join(translator, expr):
+    args = expr.op().args
+    t1, t2 = args[:2]
```

Comment:
I think these have names, like `left` and `right`. Can you use those if they exist?

Reply:
You're right! the names are left and right. thanks!